issue-creator/cronjob.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: CronJob
```

Comment:
Thanks for the explanation, it makes sense. What else does the issue creator do except creating issues?